shared/reducers/index.js

```diff
@@ -73,9 +73,9 @@ if (__DEV__) {
 }
 
 export default function (state: Object = {}, action: any): State {
-  // Reset our state if we logout
+  // Reset most of out state if we logout
   if (action.type === logoutDone) {
-    state = {}
```

Comment:
This was your change :) and it broke the logout -> login flow, so it kinda worked